ontology/tss.ttl

```diff
@@ -0,0 +1,116 @@
+@prefix tss: <http://w3id.org/rdftss/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix cdt: <http://w3id.org/awslabs/neptune/SPARQL-CDTs/> .
+@base <http://w3id.org/rdftss/> .                                                 
+                                                                  
+<http://creativecommons.org/ns#license> a owl:AnnotationProperty .
+
+dcterms:contributor a owl:AnnotationProperty .
+
+dcterms:creator a owl:AnnotationProperty .
+
+dcterms:description a owl:AnnotationProperty .
+
+dcterms:issued a owl:AnnotationProperty .
+
+dcterms:modified a owl:AnnotationProperty .
+
+dcterms:publisher a owl:AnnotationProperty .
+
+dcterms:title a owl:AnnotationProperty .
+
+vann:preferredNamespacePrefix a owl:AnnotationProperty .
+
+vann:preferredNamespaceUri a owl:AnnotationProperty .
```

Comment:
Not sure whether this is useful. Would remove these.

Reply:
I re-used this from the RML ontology of the CG.
It is basically to specify the namespace URI and prefix. Can remove it though.

Reply:
I meant all of the `a owl:AnnotationProperty` statements